stream-chat-android-compose/src/main/java/io/getstream/chat/android/compose/ui/channels/list/ChannelList.kt

```diff
@@ -240,8 +249,6 @@ public fun ChannelList(
             },
             divider = divider,
         )
-        searchQuery.query.isNotBlank() -> emptySearchContent(searchQuery.query)
-        else -> emptyContent()
```

Comment:
What about using `when` instead of nested `if`/`else`??

Something like that (Github doesn't allow me to suggest changes because git-diff contains a block of code that hasn't changed in the middle :crying_cat_face: ):
```
when {
        channels.isNotEmpty() -> Channels(
            modifier = modifier,
            contentPadding = contentPadding,
            channelsState = channelsState,
            lazyListState = lazyListState,
            onLastItemReached = onLastItemReached,
            helperContent = helperContent,
            loadingMoreContent = loadingMoreContent,
            itemContent = { itemState ->
                WrapperItemContent(
                    itemState = itemState,
                    channelContent = channelContent,
                    searchResultContent = searchResultContent,
                )
            },
            divider = divider,
        )
        isLoading -> loadingContent()
        searchQuery.query.isBlank() -> emptyContent()
        else -> emptySearchContent(searchQuery.query)
    }
```

Reply:
Love it. It's more readable 😍 
Thank you 🙌🏻